app/templates/administrative-units/administrative-unit/core-data/edit.hbs

```diff
@@ -103,32 +127,26 @@
                 </Item>
               </Grid>
             {{/if}}
-            <Grid class="au-u-margin-top-large" as |Item|>
+            <Grid as |Item|>
               <Item @labelFor="administrative-unit-scope">
                 <:label>Werkingsgebied</:label>
                 <:content>
-                  {{! TODO Make this work }}
-                  <PowerSelectMultiple
-                    @disabled={{true}}
-                    @allowClear={{true}}
-                    @searchEnabled={{true}}
-                    @loadingMessage="Aan het laden..."
-                    @noMatchesMessage="Geen resultaten"
-                    @searchMessage="Typ om te zoeken"
-                    @selected={{@model.administrativeUnit.scope}}
-                    @onChange={{fn (mut @model.administrativeUnit.scope)}}
-                    @triggerId="administrative-unit-scope"
-                    as |scope|
-                  >
-                    {{scope}}
-                  </PowerSelectMultiple>
+                  {{! TODO: Enable multiple selections and concat the selections in one label }}
```

Comment:
power select also includes a `<PowerSelectMultiple>` component which returns an array instead. The action handler can then join the items into 1 string. The tricky part is doing the reverse so the select is prefilled properly when opening the page. That probably requires a getter or a custom helper.

Reply:
The action handler also needs to create a location with the concatenated locations and put it as a label. Then it won't have a reverse part; it will be like any other location.

Reply:
I think I just don't understand how this needs to work. Creating new locations by merging other location data in to a new one sounds very weird to me.

Reply:
Initially, you have a list of municipalities and a list of provinces. For local government, there's only one municipality or province. For only worship service, we can combine many municipalities and provinces. Then at the end, we will have one scope that is the combination of locations.